app/views/developers/DeveloperDetailsView.scala.html

```diff
@@ -63,6 +63,10 @@ <h1>@developer.email</h1>
                 case false => "No"
             })</td>
         </tr>
+        <tr>
+            <th>User ID</th>
+            <td id="userID">@developer.id</td>
```

Comment:
make this UserId - we never capitalize ids as ID.  Also applies elsewhere.

Reply:
done :-)